addons/web/static/src/views/fields/properties/properties_field.scss

```diff
@@ -1,5 +1,8 @@
 .o_field_properties {
     width: 100%;
+    // be sure that the property field can take zero height
+    // in case no property exists
+    margin-bottom: 0 !important;
```

Comment:
You say "in case no property exists" but that applies all the time right?

(Maybe we already discussed this, not sure :older_man: )